app/services/concerns/xccdf/rule_group_rules.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+module Xccdf
+  # Methods related to saving RuleGroupRules
+  module RuleGroupRules
+    extend ActiveSupport::Concern
+
+    included do
+      def save_rule_group_rules
+        @required_rule_group_rules = rule_group_rules_with_requires +
+                                     rule_group_rules_with_conflicts
+        @required_rule_group_rules = @required_rule_group_rules.compact
+
+        ::RuleGroupRule.transaction do
+          ::RuleGroupRule.import!(
+            @required_rule_group_rules.select(&:new_record?), on_duplicate_key_ignore: true
```

Comment:
@skateman Using `on_duplicate_key_ignore` to check for conflicts because it'll just ignore duplicates since we don't need to update any columns.

Reply:
For consistency with other models, please use `ignore: true`, but I'm not entirely sure if we want to ignore stuff.